pkg/utils/utils.go

```diff
@@ -25,11 +25,15 @@ func BoundedIncrement(start interface{}, lower, upper int) int {
 
 // RandWalkInRange creates a new value by walking a random distance from the
 // start value.
-func RandWalkInRange(start interface{}, lower, upper int) int {
+func RandWalkInRange(start interface{}, lower, upper, step int) int {
 	// Between readings, we allow the value to change between 0 and 4.
```

Comment:
This comment appears outdated now and should probably mention the override capability